key.go

```diff
@@ -47,12 +48,117 @@ type PCP struct {
 	Context   []byte
 }
 
+// All valid cryptographic algorithms that can be used with keys.
+const (
+	KeyAlgorithmUndefined KeyAlgorithm = iota
+	AES256_GCM_SHA256
+	XCHACHA20_POLY1305
+)
+
+// KeyAlgorithm is an enum representing the algorithm
+// a cryptographic key can be used with.
+type KeyAlgorithm uint
+
+// String returns the KeyAlgorithm's string representation.
+func (a KeyAlgorithm) String() string {
+	switch a {
+	case KeyAlgorithmUndefined:
+		return "undefined"
+	case AES256_GCM_SHA256:
+		return "AES256-GCM_SHA256"
+	case XCHACHA20_POLY1305:
+		return "XCHACHA20-POLY1305"
+	default:
+		return "%" + strconv.Itoa(int(a))
```

Comment:
I am not sure I understand why '%'

Reply:
https://github.com/minio/kes/pull/312#discussion_r1071237275